server/lib/sightingsController.js

```diff
@@ -0,0 +1,28 @@
+// const mongoose = require('mongoose');
+
+const Sighting = require('../models/Sighting.js');
+
+// Create test sighting
+/* const test = Sighting({
+	animal: 'Poof',
+	location: 'Deck',
+	time: 'Now'
+}); */
+
+// Get all sightings in the database
+exports.getSightings = function () {
```

Comment:
Can we set this up with params to get specific sets of sightings now? We dont have to use it, but just set up the params so we're not changing function signatures later.

Otherwise it should be a getAllSigthings, and should probably call the getSightings that takes those options

Reply:
Added a pagination = {} and filter = {} which can then be defined later. Both are used in a console.log so that linter accepts for now, however can add in logic for both in the future.